CMakeLists.txt

```
@@ -39,17 +39,23 @@ else()
 endif()
 
 # Device specific options
-set(HOST_BACKEND_ENABLED OFF CACHE BOOL "One of the Kokkos host backend is enabled")
-if(Kokkos_ENABLE_SERIAL OR Kokkos_ENABLE_THREADS OR Kokkos_ENABLE_OPENMP)
-  set(HOST_BACKEND_ENABLED ON)
+set(TO_ENABLE_FFTW ON CACHE BOOL "Whether to enable FFTW by default")
```

```suggestion
set(TO_ENABLE_FFTW ON)
```

Is there a specific reason to make it a cache variable ?

---

Not really, I will modify accordingly